app/controllers/items.py

```diff
@@ -1,21 +1,136 @@
 from app import app, db
 from flask import Flask, request, jsonify
 from flask_sqlalchemy import SQLAlchemy
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
```

Comment:
Looks like `Flask_RESTful` extension cleaned up some syntax.  Moved the url endpoints to the `__init__.py` in `app` folder